activation-registry/pom.xml

```diff
@@ -18,7 +18,7 @@
     <parent>
         <groupId>org.eclipse.angus</groupId>
         <artifactId>angus-activation-project</artifactId>
-        <version>1.1.0-SNAPSHOT</version>
+        <version>2.0.0-SNAPSHOT</version>
```

Comment:
version change should and will be done through the extra commit/PR before this change gets merged